web/src/components/__tests__/Header.spec.tsx

```diff
@@ -40,7 +40,7 @@ describe('Header', () => {
     const { getByLabelText, getByText } = renderWithRouterAndTheme(
       <Header
         logoHref='/random_route'
-        actionItems={[<HeaderActionItemLink key={0} to='/random_route' text='random route' iconSrc='/icon.jpg' />]}
+        actionItems={[<HeaderActionItemLink key={0} to='/random_route' text='random route' icon={<div />} />]}
```

Comment:
:thought_balloon: we are not testing the icon if it's in the document but this is ok for me.